packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -296,8 +321,8 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
 
     const SingleActivator(LogicalKeyboardKey.arrowLeft, shift: true, meta: true): const ExpandSelectionToLineBreakIntent(forward: false),
     const SingleActivator(LogicalKeyboardKey.arrowRight, shift: true, meta: true): const ExpandSelectionToLineBreakIntent(forward: true),
-    const SingleActivator(LogicalKeyboardKey.arrowUp, shift: true, meta: true): const ExtendSelectionToDocumentBoundaryIntent(forward: false, collapseSelection: false),
-    const SingleActivator(LogicalKeyboardKey.arrowDown, shift: true, meta: true): const ExtendSelectionToDocumentBoundaryIntent(forward: true, collapseSelection: false),
+    const SingleActivator(LogicalKeyboardKey.arrowUp, shift: true, meta: true): const ExpandSelectionToDocumentBoundaryIntent(forward: false),
+    const SingleActivator(LogicalKeyboardKey.arrowDown, shift: true, meta: true): const ExpandSelectionToDocumentBoundaryIntent(forward: true),
```

Comment:
Why was `collapseSelection: false` removed?

Reply:
Because on macOS the correct action is `ExpandSelectionToDocumentBoundaryIntent`, not `ExtendSelectionToDocumentBoundaryIntent`. And `ExpandSelectionToDocumentBoundaryIntent` has no `collapseSelection` property.

Note that cmd+shift+up on macOS is same as shift+home, they both result in `moveToBeginningOfDocumentAndModifySelection:` so the action needs to be same. shift+home already had the correct action (Expand).